### ISSUE TITLE
Stop readout time stamp equals start readout time stamp

### ISSUE THREAD

**Issue by DavidLP** [SiLab-Bonn/pyBAR#48]
It is possible that in the raw meta data start and stop readout time stamp are equal. That does not make sense.

![capture](https://cloud.githubusercontent.com/assets/6951141/14409027/4495cbfe-ff0b-11e5-9ec8-865b62e5a130.PNG)

**Comment by laborleben** [comment]
This should not happen. But this might be a problem of the system time.

**Comment by DavidLP** [comment]
Is it fixed or understood?

**Comment by laborleben** [comment]
No, so I need more information. The fix will just mask the error and it will not show up.

**Comment by laborleben** [comment]
To be more clear, it would be better if you revert the fix, otherwise I will miss the condition that triggers it.

**Comment by laborleben** [comment]
Ah you already made a new interpreter version. OK, then I will ignore it. I leave it up to you to fix it.

**Comment by DavidLP** [comment]
Since we do not rely on the time stamp anywhere, we just leave this bug report open until somebody needs to fix it.
